F/finufft/build_tarballs.jl

```diff
@@ -36,7 +39,7 @@ install -Dvm 0755 "lib/libfinufft.${dlext}" "${libdir}/libfinufft.${dlext}"
 
 # Expand for microarchitectures on x86_64 (library doesn't have CPU dispatching)
 # Tests on Linux/x86_64 yielded a slow binary with avx512 for some reason, so disable that
-platforms = expand_cxxstring_abis(expand_microarchitectures(supported_platforms(), ["x86_64", "avx", "avx2"]); skip=!Sys.iswindows)
+platforms = expand_cxxstring_abis(expand_microarchitectures(supported_platforms(), ["x86_64", "avx", "avx2", "avx512"]); skip=!Sys.iswindows)
```

Comment:
This won't work because of https://github.com/JuliaPackaging/BinaryBuilder.jl/pull/1194 and https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/233

Reply:
Oh, well, it works precisely because you disabled avx512 instructions.